studio/components/interfaces/BillingV2/Subscription/CostControl/SpendCapSidePanel.tsx

```diff
@@ -85,9 +85,7 @@ const SpendCapSidePanel = () => {
     }
   }
 
-  const billingMetricCategories = Array.from(
-    new Set(BILLING_BREAKDOWN_METRICS.map((it) => it.category))
-  )
+  const billingMetricCategories = ['database', 'auth', 'storage', 'realtime', 'edge-functions']
```

Comment:
random strings?

Reply:
Can't be typed as pricing.json is not typed (due to being JSON), see other comment for possible refactor